gen_cert.py

```diff
@@ -1274,6 +1125,22 @@ def _generate_v4_certificate(
             achievements_string = grade_interstitial.decode('utf-8').format(grade=grade_html) + '<br /><br />'
         achievements_paragraph = u"{0}{1}".format(achievements_string, achievements_description_string)
 
+        # Overide achievements/interstitial strings with yaml designations info based on designation
+        designation_tag = ''
+        if designation:
+            # Add student name designation if it exists
+            student_name = u"{name}, {designation}".format(
+                name=student_name,
+                designation=designation.decode('utf-8'),
+            )
+            if 'designations' in self.cert_data:
+                for key, value in self.cert_data.get('designations').iteritems():
+                    if designation in value['titles']:
+                        achievements_string = value['credits']
```

Comment:
Same as above, titles and credits are currently connected.  I can fix this if we determine that it's broken.  Probably in a new commit since it would be new functionality.